packages/lab/src/dialog/DialogTitle.tsx

```diff
@@ -42,19 +62,22 @@ export const DialogTitle = ({
   const status = statusProp ?? (statusContext as ValidationStatus);
 
   return (
-    <H2
+    <div
       className={clsx(
         withBaseName(),
         {
           [withBaseName("withAccent")]: !disableAccent && !status,
-          [withBaseName(status)]: !!status,
+          [withBaseName(status)]: !status,
         },
         className
       )}
       {...rest}
     >
       {status && <StatusIndicator status={status} />}
-      {children}
```

Comment:
children removed - if and additional component is needed e.g Step tracker this can be done by wrapping Dialog title in a Layout Component. See the Large Dialog within the size examples

Reply:
Why do you need to remove children, which is a breaking change, and not instead just introduce one prop `subtitle`?